File: app/src/main/java/com/whaleshark/basicarch/binding/BindingAdapters.java
package com.whaleshark.basicarch.binding;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

/**
 * @author stipton
 */

public class BindingAdapters {

    @BindingAdapter("imageUrl")
    public static void bindImage(ImageView imageView, String url) {
        Glide.with(imageView).load(url).into(imageView);
    }
}

File: app/src/main/java/com/whaleshark/basicarch/binding/BindingAdapters.kt
package com.whaleshark.basicarch.binding

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.bumptech.glide.Glide

/**
 * @author stipton
 */

object BindingAdapters {

    @BindingAdapter("imageUrl")
    fun bindImage(imageView: ImageView, url: String) {
        Glide.with(imageView).load(url).into(imageView)
    }
}
